docs/recipes/README.md

```diff
@@ -0,0 +1,9 @@
+# 📄 Recipes for workflows with semantic-release
+
+* CI configuration
```

Comment:
Maybe it would be better to use heading in the readme?
Something like this:
```
## CI configuration
### [Travis CI build stages](./travis_build_stages.md)
```

Reply:
I don't think that would look better, because the headings would have no indentation when rendered. This is intended to be a ToC

Reply:
Comment to be ignored. Let's keep it that way.

Reply:
Ok